File: CryptoCurrency-Resources/API-Calls/Coin-Market-Cap.py
#!/usr/bin/python

import requests
Session
from requests.exceptions import ConnectionError, Timeout, TooManyRedirects
import json

url = 'https://pro-api.coinmarketcap.com/v1/cryptocurrency/listings/latest'
parameters = {
  'start':'1',
  'limit':'5000',
  #'symbol':'BTC,ETH,USDT, USDC, BNB, BUSD, XRP, ADA, SOL, DOGE'
  'convert':'USD'
}
headers = {
  'Accepts': 'application/json',
  'X-CMC_PRO_API_KEY': 'd560b425-b17a-4cba-8f05-8fd111a19be0',
}

session = Session()
session.headers.update(headers)

try:
  response = session.get(url, params=parameters)
  data = json.loads(response.text)
  print(data)
except (ConnectionError, Timeout, TooManyRedirects) as e:
  print(e)

File: CryptoCurrency-Resources/API-Calls/Coin-Market-Cap.cs
using System.Net;
using System.Web;

// Fetches all active cryptocurrencies by market cap and returns market values in USD

class AllActiveCurrenciesMarketValuesInUSD
{
  private static string API_KEY = "d560b425-b17a-4cba-8f05-8fd111a19be0";

  public static void Main(string[] args)
  {
    try
    {
    Console.WriteLine(makeAPICall());
    }
    catch (WebException e)
    {
    Console.WriteLine(e.Message);
    }
  }

  static string makeAPICall()
  {
    var URL = new UriBuilder("https://pro-api.coinmarketcap.com/v1/cryptocurrency/listings/latest");

    var queryString = HttpUtility.ParseQueryString(string.Empty);
    queryString["start"] = "1";
    queryString["limit"] = "5000";
    queryString["convert"] = "USD";

    URL.Query = queryString.ToString();

    var client = new WebClient();
    client.Headers.Add("X-CMC_PRO_API_KEY", API_KEY);
    client.Headers.Add("Accepts", "application/json");
    return client.DownloadString(URL.ToString());

  }

}